openassessment/xblock/static/js/src/studio/oa_container_item.js

```diff
@@ -109,7 +117,16 @@ OpenAssessment.Prompt.prototype = {
      **/
     description: function(text) {
         var sel = $('.openassessment_prompt_description', this.element);
-        return OpenAssessment.Fields.stringField(sel, text);
+        if (this.kwargs.addWysiwyg) {
```

Comment:
You shouldn't have the prototype of the class introspecting for a subclass's field ("addWysiwyg"). Why not override "description" in the Prompt class rather than modify "description" in the prototype?